### ISSUE TITLE
Itron Actaris Schlumberger

### ISSUE THREAD

**Issue by Y0rk-code** [martijnvwezel/watermeter-esphome#41]
Is there a 3d case to make it fit on the Itron Actaris Schlumberger?

**Comment by martijnED** [comment]
No, because I do not have my own I cannot design it. You you need help like a 3d model of the water meter?

**Comment by shuffleznl** [comment]
Fitted to our Itron Aquadis+ (which seems incredibly similar) with 3 tie-wraps and some double-sided tape. Seems to seat quite security between the 'stand-offs' where the blue cap normally rests on.

**Comment by martijnED** [comment]
Do you mind sharing a picture?

**Comment by shuffleznl** [comment]
Not entirely sure it's exactly over the right area, initially it measured (seemingly) too high values (or we had some insane water usage :)). Maybe I'll reset the values in HA, or try to recalibrate (if I can figure that out). 
![PXL_20240412_202150106](https://github.com/martijnvwezel/watermeter-esphome/assets/8324786/975268a7-2c61-4335-bf3e-04b21fd1ae0c)
![PXL_20240412_202136291](https://github.com/martijnvwezel/watermeter-esphome/assets/8324786/2d675a99-a4f1-40a6-8274-4925d4843632)
![PXL_20240412_202123480](https://github.com/martijnvwezel/watermeter-esphome/assets/8324786/b5e1821b-3f67-4c8d-a5cb-11a282550b97)

If you're up for it can perhaps scan the unit (photogrammetry) to co-develop a PCB-mount/adapter.

**Comment by shuffleznl** [comment]
Some top-down views from other sources:
https://cdn.store-factory.com/www.compteur-energie.com/content/product_11170581b.png?v=1607702126
https://www.metermarket.co.uk/assets/products/FullSizeRemanifoldnder.jpg
https://forum.mysensors.org/uploads/files/1449418299492-aquadis.jpeg

Angled view for good measure
https://www.metermarket.co.uk/assets/products/_ogImage/FullSizeReITRON-AQUADISnder.jpg

Happy to print & test any prototype mount.

**Comment by martijnvwezel** [comment]
Very nice looking!
![image](https://github.com/martijnvwezel/watermeter-esphome/assets/4363788/a86688c4-d23c-42d5-bec7-0ac8ecc13a8d)


maybe we can create some 3D print that can click in those 3 slots

**Comment by shuffleznl** [comment]
Will try to design a basic (like Tinkercad, or perhaps Plasticity basic) 2D/3D model when I get around to it. Shouldnt be too hard and at least help with alignment (e.g. now reading declining values again, perhaps tiewraps shifted. Or Muino accounts for the rain water accumulation (rofl) ).

**Comment by martijnED** [comment]
Hahaha very nice hope you can fix it!
